apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/TestRunner.ts

```diff
@@ -127,11 +128,12 @@ export class TestRunner {
       buildSuite,
       testCases: [],
       nestingLevel: (this._currentTestSuite?.nestingLevel || 0) + 1,
+      // eslint-disable-next-line no-unneeded-ternary
```

Comment:
Why do we disable that? I assume eslint wants us to use nullish coalescent operator here?

Reply:
I disable it because this was already handled in https://github.com/software-mansion/react-native-reanimated/pull/6047